ci/docker.sh

```diff
@@ -37,32 +37,34 @@ if [ -z "${docker_hub_username:-}" ]; then
   fn_die "ERROR: DOCKER_HUB_USERNAME variable is not set. Exiting ..."
 fi
 
-docker_tag=""
+docker_tag_full=""
 if [ "${is_a_release}" = "true" ]; then
-  docker_tag="${github_ref_name}"
+  docker_tag_full="${github_ref_name}"
 fi
 
 # Building and publishing docker image
-if [ -n "${docker_tag:-}" ]; then
-  log_info "=== Building Docker image: ${docker_hub_org}/${docker_image_build_name}:${docker_tag} ==="
-  docker build --build-arg PROFILE=production -f "${docker_file_path}" -t "${docker_hub_org}/${docker_image_build_name}:${docker_tag}" .
+if [ -n "${docker_tag_full:-}" ]; then
+  log_info "=== Building Docker image: ${docker_hub_org}/${docker_image_build_name}:${docker_tag_full} ==="
+  docker build --build-arg PROFILE=production -f "${docker_file_path}" -t "${docker_hub_org}/${docker_image_build_name}:${docker_tag_full}" .
 
   # Publishing to DockerHub
   log_info "=== Publishing Docker image(s) on Docker Hub ==="
   echo "${docker_hub_token}" | docker login -u "${docker_hub_username}" --password-stdin
 
   # Docker image(s) tags for PROD vs DEV release
   if [ "${prod_release}" = "true" ]; then
-    publish_tags=("${docker_tag}" "latest")
+    docker_tag_node=$(cut -d'-' -f1 <<< $docker_tag_full)
```

Comment:
The idea is to publish 2 tags if it comes in the format of 1.1.1-2.2.2? So we want to publish 1.1.1-2.2.2 as well as 1.1.1?

Reply:
The answers to your questions are both yes.
Modified as per your suggestion.